e2e/container/container_test_other.go

```diff
@@ -0,0 +1,15 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
what is other mean?

Reply:
It means "not Linux". `getCGroupMode` doesn't really make sense unless you're running on Linux